drivers/bmp180/bmp180.c

```diff
@@ -159,7 +159,7 @@ uint32_t bmp180_read_pressure(const bmp180_t *dev)
     x1 = (p >> 8) * (p >> 8);
     x1 = (x1 * 3038) >> 16;
     x2 = (-7357 * p) >> 16;
-
+    
```

Comment:
Drop this added white space

Reply:
I didnt add this white space. It was already there, as you can see in the bmp180.c file that is currently in the master-branch. As you can see i didnt add an additional white space. I added one and removed one

Reply:
I'm afraid you did. But I see that you already removed it in 65e1a33e. Great.